src/lib/features/feature-toggle/types/feature-toggle-store-type.ts

```diff
@@ -36,6 +37,10 @@ export interface IFeatureToggleStore extends Store<FeatureToggle, string> {
         userId?: number,
         archived?: boolean,
     ): Promise<FeatureToggle[]>;
+    getPlaygroundFeatures(
+        dependentFeaturesEnabled: boolean,
```

Comment:
why are we passing it as args instead of passing flag in a constructor as we did before in the previous client store?

Reply:
I didn't want to introduce it in the feature toggle store, because the feature toggle store does not currently receive the config. I decided to go this route for now, since I expect the feature flag to be removed shortly.